### ISSUE TITLE
Testing version Version 2-20160519

### ISSUE THREAD

**Issue by paaljoachim** [obenland/shiny-updates#124]
**Plugins**
Add new screen
Clicked three plugins to get them installed. Works nicely. 

Installed screen:
I am noticing the fresh update yellow and orange circle arrows. Looks nice!
Really nice compared to the old whitish background and red circle arrows.
The green updated bar might be just a little too white-green. Compared to the yellow the green has too much white in it.

**Delete** brings up the plain popup box mentioning the name of the plugin that is to be deleted. Looks not as nice. Could perhaps use a red bar instead of the grey in the top of the popup box to brighten the color a bit. Works nicely.

**Bulk delete**. Deletes without warning whatever plugins that are selected. Works nicely.
Should Bulk delete also have a popup box like the normal one plugin delete mentioning the plugins that will be deleted?

**Themes**
Add Themes screen
Clicking Install --- blue installing button. That end up with an Installed green bar at the top. A little different from how installing a plugin looks but that is alright. 

Installed screen:
Yellow bar signals that an update is available above installed themes. Perhaps a tiny grey-ish 1px bottom border in the yellow update notice? As it would create a border between the update bar and the theme screenshot.
Same goes for green updated (and in the Add Themes Installed green bar) bar. 

Updates screen
Nice and clean section! A huge improvement over existing updates screen!
Should there be a difference in background with updating Core, plugins and themes? Or a thin border between the sections. To create a little more difference between the three main areas? I have a feeling something is still missing here....I can not put my finger on it yet.
The text: "If you need to re-install version 4.5.2, you can do so here."
What about something like: Click the button if you need to reinstall version 4.5.2.

Updating.
Should color be added to the button?

Auto update. 
Any plans on adding Auto update of core, plugins and themes?
For instance adding checkboxes on the bottom of the screen to auto update. Which was included sometime ago but has now been removed.

What else should I look at?

Bottom line.
I am very impressed by the nice and shiny sections!!

**Comment by swissspidy** [comment]
Thanks again for taking the time to test the plugin and your valuable feedback!

Just picking some statements out of it:

**Plugin Installs**

> The green updated bar might be just a little too white-green. Compared to the yellow the green has too much white in it.

I think that's the regular admin notice used elsewhere in the admin area as well. Not sure how much we can change there.

**Plugin Deletions**

> Brings up the plain popup box mentioning the name of the plugin that is to be deleted. Looks not as nice. Could perhaps use a red bar instead of the grey in the top of the popup box to brighten the color a bit.

It's a regular `window.confirm()` dialog, which used elsewhere in core as well, e.g. when deleting themes. There were some discussions about using custom modal dialogs for this on Trac, which ended in a _maybelater_. That means it's definitely out of scope for this plugin, but could be improved in a future WordPress release.

**Bulk Plugin Deletions**

> Deletes without warning whatever plugins that are selected. Should Bulk delete also have a popup box like the normal one plugin delete mentioning the plugins that will be deleted?

Not having a warning for bulk deletions is indeed kinda unexpected. @obenland What do you think?

**Theme Installs**

> Yellow bar signals that an update is available above installed themes. Perhaps a tiny grey-ish 1px bottom border in the yellow update notice? As it would create a border between the update bar and the theme screenshot. Same goes for green updated bar.

Here's how it would look like with the same border (actually a box-shadow) as used for the rest of the card:

<img width="407" alt="screen shot 2016-05-20 at 11 38 00" src="https://cloud.githubusercontent.com/assets/841956/15427842/087003a4-1e98-11e6-8a5c-3604be75cf19.png">


**Updates screen**

> Should there be a difference in background with updating Core, plugins and themes? Or a thin border between the sections. To create a little more difference between the three main areas?

I tried having a different background color for core and the rest, but it doesn't look _that_ nice. At least there's now a border between sections:

<img width="1264" alt="screen shot 2016-05-20 at 10 59 09" src="https://cloud.githubusercontent.com/assets/841956/15422657/eb97c50a-1e79-11e6-9622-6c0c3f44320e.png">

> What about something like: Click the button if you need to reinstall version 4.5.2.

That wouldn't be really good for accessibility. The text used right now is the same as in WordPress core and I think we could keep it without problems.

We could do a general review of all text strings though.

**General**

> Updating: Should color be added to the button?

Good question. I'll leave that for the design team to answer as I'm not sure how that'd look like.

**Auto updates**

> Any plans on adding Auto update of core, plugins and themes?

Not as part of Shiny Updates. Citing 6eb4b05758d3726919d1ce6652eafbbc2fd5992b:

Auto updates are outside of the scope of this plugin. There are many more challenges around that that this plugin is not prepared to solve.

**Comment by swissspidy** [comment]
I now created a new issue for improving plugin deletions, see #127.

**Comment by paaljoachim** [comment]
Thanks for the feedback Pascal!
